ee-9/ee-9-api/src/main/resources/modules/system/layers/base/jakarta/activation/api/main/module.xml

```diff
@@ -25,17 +25,12 @@
 <module xmlns="urn:jboss:module:1.9" name="jakarta.activation.api">
 
     <resources>
-        <artifact name="${com.sun.activation:jakarta.activation}"/>
+        <artifact name="${jakarta.activation:jakarta.activation-api}"/>
     </resources>
 
     <dependencies>
         <module name="java.logging" />
-        <module name="javax.api" />
-        <module name="javax.mail.api" optional="true">
-            <imports><include path="META-INF"/></imports>
-        </module>
-        <module name="com.sun.xml.messaging.saaj" optional="true"/>
-        <module name="org.jboss.ws.native.jbossws-native-core" optional="true"/>
-        <module name="org.apache.cxf" optional="true"/>
```

Comment:
@jimma Please review these last 3 lines.  The org.jboss.ws.native.jbossws-native-core module doesn't exist, so removing that one for sure won't break anything. :)

Reply:
@yersan The WF Preview ts failures make me suspect this needs some other optional deps so it can find implementations.

Reply:
Right @bstansberry , I've added the implementation to the modules that export it to CXF, let's see how it goes.

About the removals I did on this module, I still need to understand the original intention of having them there, but I believe they are unnecessary at least for the API, because the Jarkarta Activation API 2.1 doesn't require them.
It seems they could be providers for the Jakarta Activation SPI, I hope @jimma can shed some light there.

About the Jakarta Mail one, I haven't found the reason why it was there yet; this PR is still in progress. Any lights are welcome :)

Let's see what CI says.

Reply:
@yersan @bstansberry  The "com.sun.xml.messaging.saaj" and "jbossws-native-core" dependencies are added for jaxrpc. We should remove them all. Please see [AS7-2550](https://issues.redhat.com/browse/AS7-2550).
For "org.apache.cxf" dependency, I found this is added for [AS7-2836](https://issues.redhat.com/browse/AS7-2836) and it's quite old change. But I think "org.apache.cxf" can be safely removed too. The activiation api should not depend any thing from CXF.